docs/HowToFAQ/FAQFunctionUnroll0.dfy

```diff
@@ -0,0 +1,27 @@
+function LengthSum(s: seq<string>): nat {
```

Comment:
Could we simplify the example to use just `Sum` instead of `LengthSum`?

Reply:
Done

Reply:
I mean Sum of numbers, not of lengths